matrix_sdk/src/request_builder.rs

```diff
@@ -1,20 +1,22 @@
-use crate::api;
-use crate::events::room::power_levels::PowerLevelsEventContent;
-use crate::events::EventJson;
-use crate::identifiers::{DeviceId, RoomId, UserId};
-use api::r0::account::register;
-use api::r0::account::register::RegistrationKind;
-use api::r0::directory::get_public_rooms_filtered::{self, Filter, RoomNetwork};
-use api::r0::filter::RoomEventFilter;
-use api::r0::membership::Invite3pid;
-use api::r0::message::get_message_events::{self, Direction};
-use api::r0::room::{
-    create_room::{self, CreationContent, InitialStateEvent, RoomPreset},
-    Visibility,
+use std::convert::TryFrom;
+
+use matrix_sdk_common::{
+    api::r0::{
+        account::register::{self, RegistrationKind},
+        directory::get_public_rooms_filtered::{self, Filter, RoomNetwork},
+        filter::RoomEventFilter,
+        membership::Invite3pid,
+        message::get_message_events::{self, Direction},
+        room::{
+            create_room::{self, CreationContent, InitialStateEvent, RoomPreset},
+            Visibility,
+        },
+        uiaa::AuthData,
+    },
```

Comment:
Do you have a preference for import format?

Reply:
Nesting them like Ruma does is fine, prehaps even preferred, but I don't think I want to enforce a style.